testdata/src/main/java/com/hedvig/app/testdata/feature/embark/builders/PassageBuilder.kt

```diff
@@ -34,6 +35,7 @@ data class PassageBuilder(
         redirects = redirects,
         action = action,
         api = api,
-        allLinks = links.map { EmbarkStoryQuery.AllLink(fragments = EmbarkStoryQuery.AllLink.Fragments(it)) }
+        allLinks = links.map { EmbarkStoryQuery.AllLink(fragments = EmbarkStoryQuery.AllLink.Fragments(it)) },
+        externalRedirect = EmbarkStoryQuery.ExternalRedirect(data = EmbarkStoryQuery.Data1(location = EmbarkExternalRedirectLocation.OFFER))
```

Comment:
Won't this make every passage redirect to OFFER, no matter what? I think we might want this to be configurable in the builder, right? 🤔

Reply:
Also not included when I selected the correct base branch